root/user/components/UserTagFilters.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * @flow strict
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import FormRow from '../../components/FormRow';
+
+type PropsT = {
+  +$c: CatalystContextT,
+  +showDownvoted: boolean,
+  +showSortSelect?: boolean,
+  +showVotesSelect?: boolean,
+  +sortBy?: 'count' | 'countdesc' | 'name',
+};
+
+const InlineSubmitButton = (
+  <>
+    {' '}
+    <button type="submit">
+      {l('Submit')}
+    </button>
+  </>
+);
```

Comment:
`l('Submit')` won't work here, so better to either declare this inside `UserTagFilters` or make it an actual component.

(Not sure if we have an eslint rule for this, but if we keep it declared as an inline element, I think `inline` should start with a lowercase letter since it's neither a class or component.)

Reply:
I changed it to a component and checked it does work in French.